packages/elements/code-block/src/transforms/insertCodeBlock.ts

```diff
@@ -49,6 +49,7 @@ export const insertCodeBlock = (
     {
       type: getPlatePluginType(editor, ELEMENT_CODE_BLOCK),
       children: [],
+      lang: '',
```

Comment:
I think undefined is probably better than an empty string. It can be an option of the function if needed.